.hooks/pre-commit

```diff
@@ -2,6 +2,19 @@
 
 # validating that each modified file has a valid schema, release notes, proper prefix & suffix
 echo "Validating files..."
+DO_DEACTIVATE=""
+DO_ACTIVATE=""
+if [[ -n "${VIRTUAL_ENV}" && "${VIRTUAL_ENV}" != "${PWD}/venv" && -e "${VIRTUAL_ENV}/bin/activate" ]] && command -v deactivate >/dev/null 2>&1; then
+    # we are in a vritual env. Deactivate and store the previous VIRTUAL_ENV
+    echo "Deactivating current virtual env: $VIRTUAL_ENV"
+    DO_ACTIVATE="${VIRTUAL_ENV}"
+    source deactivate
+fi
+if [ -e "./venv/bin/activate" -a -z "${VIRTUAL_ENV}" ]; then
+    echo "Activating Demisto Content virtual env..."
+    source ./venv/bin/activate
+    DO_DEACTIVATE=1
+fi
 if [[ -z "${WINDIR}" ]]
     then
         PYTHONPATH="`pwd`:${PYTHONPATH}" python Tests/scripts/validate_files.py -t true
```

Comment:
File Tests/scripts/validate_files.py using /Tests/scripts/hook_validations/secrets.py, the secrets.py is python2 version due to iteritems() on dict (line294) (without shebang or any version declaration), so when calling the script inside the environment it will fail.